robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -110,10 +172,20 @@ public void setSnapshotsEnabled(boolean enabled) {
         }
     }
 
+    /*
+     * Set which pipeline you want to use (0-9)
```

Comment:
Start the sentence with `Sets` instead of `Set`

Reply:
done